src/autopas/utils/logging/IterationMeasurements.h

```diff
@@ -37,23 +37,22 @@ struct IterationMeasurements {
   bool energyMeasurementsPossible{false};
 
   /**
-   * Energy consumed by the entire system.
+   * Total energy consumed in watts.
```

Comment:
Shouldn't this be the average energy consumed per time in Watts? Or am I missing something?